api/src/test/java/org/openmrs/module/biometric/api/service/impl/ConfigServiceTest.java

```diff
@@ -594,10 +587,12 @@ public void retrieveAllConfigUpdates_shouldThrowExceptionWhenAddressHierarchyIsN
       throws BiometricApiException, IOException {
     String configGp = "12345";
     given(
-        Context.getAdministrationService().getGlobalProperty(BiometricApiConstants.MAIN_CONFIG_GP))
+            Context.getAdministrationService()
+                .getGlobalProperty(BiometricApiConstants.MAIN_CONFIG_GP))
         .willReturn("12345");
     given(
-        Context.getAdministrationService().getGlobalProperty(BiometricApiConstants.LOCALIZATION_GP))
+            Context.getAdministrationService()
+                .getGlobalProperty(BiometricApiConstants.LOCALIZATION_GP))
```

Comment:
Makes sense  . What if this global property is not configured. what will happen

Reply:
this is only formatting change. and this is only test where we define what we want to return when Context.getAdministrationService().getGlobalProperty(BiometricApiConstants.LOCALIZATION_GP) is called